onnxscript/tests/function_libs/torch_lib/extra_opinfo.py

```diff
@@ -1461,24 +1461,86 @@ def shape(size, rank, with_batch_channel=True):
 
     for align_corners in align_corners_options:
         yield opinfo_core.SampleInput(
-            make_arg(shape(D, rank)), shape(S, rank, False), align_corners
+            make_arg(shape(D, rank)), shape(S, rank, False), align_corners=align_corners
         )
         yield opinfo_core.SampleInput(
             make_arg(shape(D, rank)),
             shape(L, rank, False),
-            align_corners,
+            align_corners=align_corners,
```

Comment:
align_corners should not be kwargs.

Likely the reason why test failed on False. It is supposed to be arg and now there is no wrangler to move it from kwargs to args.

Reply:
I see. Let me fix this